editor/animation_track_editor.cpp

```diff
@@ -4903,6 +4906,8 @@ void AnimationTrackEditor::_notification(int p_what) {
 
 			const int track_separation = get_theme_constant(SNAME("track_v_separation"), SNAME("AnimationTrackEditor"));
 			track_vbox->add_theme_constant_override("separation", track_separation);
+
+			function_name_toggler->add_theme_color_override("icon_pressed_color", Color("GRAY"));
```

Comment:
```suggestion
			function_name_toggler->add_theme_color_override("icon_pressed_color", get_theme_color("icon_disabled_color");
```

Reply:
I tried using `get_theme_color` but, no matter what color you pick, it yields this same result when toggled, which looks out of place.

![image](https://github.com/user-attachments/assets/115c441e-5d83-4748-a6cd-a4b48629991b)

Reply:
Oh sorry, try
```
function_name_toggler->add_theme_color_override("icon_pressed_color", get_theme_color("icon_disabled_color", EditorStringName(Editor));
```

Reply:
Now it is working as intended:
![Changes3](https://github.com/user-attachments/assets/dd33c05b-f8eb-4ee7-91ef-1cba8dc221f3)